From 75f04c14e49e084e41bdd5491edad8823773a08c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20K=C4=99pie=C5=84?= <github@kempniu.pl>
Date: Tue, 14 Feb 2017 12:42:04 +0100
Subject: [PATCH 40/98] Ensure compatibility with gtk-vnc 0.7.0+

---
 libvncserver/websockets.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/libvncserver/websockets.c b/libvncserver/websockets.c
index 72396c2..0b2d46f 100644
--- a/libvncserver/websockets.c
+++ b/libvncserver/websockets.c
@@ -245,7 +245,10 @@ webSocketsCheck (rfbClientPtr cl)
       return FALSE;
     }
 
-    if (strncmp(bbuf, "<", 1) == 0) {
+    if (strncmp(bbuf, "RFB ", 4) == 0) {
+        rfbLog("Normal socket connection\n");
+        return TRUE;
+    } else if (strncmp(bbuf, "<", 1) == 0) {
         rfbLog("Got Flash policy request, sending response\n");
         if (rfbWriteExact(cl, FLASH_POLICY_RESPONSE,
                           SZ_FLASH_POLICY_RESPONSE) < 0) {
-- 
2.9.4

